### DISCUSSION TITLE
New Dashboard with Animation

### DISCUSSION THREAD

**Discussion by jasonacox** [jasonacox/Powerwall-Dashboard#26] [Announcements]
The pyPowerwall Proxy can now pull the live power flows from the Powerwall web portal and embed those in iFrames (thanks to @danisla for the PR). I have created a Dashboard that includes this live animation.  Is this useful?  What do you think?

I'm including screenshots here and the dashboard JSON that you can import:

<img width="950" alt="image" src="https://user-images.githubusercontent.com/836718/169719145-ad7d6266-8db6-493a-b393-82abcad4e106.png">

Dashboard JSON: https://github.com/jasonacox/Powerwall-Dashboard/blob/main/dashboard-animation.json

_NOTE: This requires an update (run `upgrade.sh` to get latest pyPowerwall container and Grafana settings to display the animation iFrame).  It also assumes that your pypowerwall proxy is running on the same host as your Grafana dashboard._

**Comment by jasonacox** [comment]
Updated to include Grid Status time series graph instead. 

<img width="899" alt="PowerwallDashboard-animation-grid" src="https://user-images.githubusercontent.com/836718/173971313-11ede1ea-8ed6-4750-8404-b57947723355.png">

**Reply by cwagz** [reply]
Here is a grab from mine. I have a lot of equipment running 24x7.

<img width="964" alt="PowerwallDash" src="https://user-images.githubusercontent.com/13752647/177013062-2a05d9a0-b61b-4fe2-b600-90b52204c000.png">

<img width="967" alt="PowerwallDash2" src="https://user-images.githubusercontent.com/13752647/177013176-17623de4-e7e1-4ad6-adbb-e6393593ed8c.png">
